src/toil/jobStores/abstractJobStore.py

```diff
@@ -866,3 +884,28 @@ def _defaultTryCount(self):
     @classmethod
     def _validateSharedFileName(cls, sharedFileName):
         return bool(cls.sharedFileNameRegex.match(sharedFileName))
+
+
+class SupportJobStore(AbstractJobStore):
+    @classmethod
+    def _supportsUrl(cls, url, mode=mode.both):
+        def importOnly():
+            return url.scheme.lower() in ('http', 'https')
+
+        def exportOnly():
+            return False
+
+        def both():
+            return False
+
+        return cls._supportsUrlBody(mode, importOnly, exportOnly, both)
```

Comment:
I agree, this is a lot of code for what looks to me like more of an edge case. If these changes are reversed and say this happens
```
with Toil(options) as toil:
    toil.exportFile(someJobStoreFileID, 'http://someURL"')
```
A not implemented error will be raised because jobStoreSupport() lacks ``` _writeToUrl()```. Since this exception will be the one ultimately seen by the user, what do you think of raising a more specific exception than not implemented, something along the lines of ``` UnsupportedExportError```, assuming also _readFromUrl would raise ``` UnsuportedImportError```